style.mss

```diff
@@ -2,26 +2,335 @@ Map {
+                "Noto Sans Glagolitic Bold", "Noto Sans Gothic Bold",
+                "Noto Sans Gujarati UI Bold", "Noto Sans Gujarati Bold",
+                "Noto Sans Gurmukhi UI Bold", "Noto Sans Gurmukhi Bold",
+                "Noto Sans Hanunoo Bold", "Noto Sans Hebrew Bold",
+                "Noto Sans Imperial Aramaic Bold",
+                "Noto Sans Inscriptional Pahlavi Bold",
+                "Noto Sans Inscriptional Parthian Bold",
+                "Noto Sans Javanese Bold", "Noto Sans Kaithi Bold",
+                "Noto Sans Kannada UI Bold", "Noto Sans Kannada Bold",
+                "Noto Sans Kayah Li Bold", "Noto Sans Kharoshthi Bold",
+                "Noto Sans Khmer UI Bold", "Noto Sans Khmer Bold",
+                "Noto Sans Lao UI Bold", "Noto Sans Lao Bold",
+                "Noto Sans Lepcha Bold",
+                "Noto Sans Limbu Bold", "Noto Sans Linear B Bold",
+                "Noto Sans Lisu Bold", "Noto Sans Lycian Bold",
+                "Noto Sans Lydian Bold",
+                "Noto Sans Malayalam UI Bold", "Noto Sans Malayalam Bold",
+                "Noto Sans Mandaic Bold", "Noto Sans Meetei Mayek Bold",
+                "Noto Sans Mongolian Bold",
+                "Noto Sans Myanmar UI Bold", "Noto Sans Myanmar Bold",
+                "Noto Sans New Tai Lue Bold", "Noto Sans NKo Bold",
+                "Noto Sans Ogham Bold", "Noto Sans Ol Chiki Bold",
+                "Noto Sans Old Italic Bold", "Noto Sans Old Persian Bold",
+                "Noto Sans Old South Arabian Bold",
+                "Noto Sans Old Turkic Bold",
+                "Noto Sans Oriya UI Bold", "Noto Sans Oriya Bold",
+                "Noto Sans Osmanya Bold", "Noto Sans Phags Pa Bold",
+                "Noto Sans Phoenician Bold", "Noto Sans Rejang Bold",
+                "Noto Sans Runic Bold", "Noto Sans Samaritan Bold",
+                "Noto Sans Saurashtra Bold", "Noto Sans Shavian Bold",
+                "Noto Sans Sinhala Bold", "Noto Sans Sundanese Bold",
+                "Noto Sans Syloti Nagri Bold", "Noto Sans Symbols Bold",
+                "Noto Sans Syriac Eastern Bold",
+                "Noto Sans Syriac Estrangela Bold",
+                "Noto Sans Syriac Western Bold", "Noto Sans Tagalog Bold",
+                "Noto Sans Tagbanwa Bold", "Noto Sans Tai Le Bold",
+                "Noto Sans Tai Tham Bold", "Noto Sans Tai Viet Bold",
+                "Noto Sans Tamil UI Bold", "Noto Sans Tamil Bold",
+                "Noto Sans Telugu UI Bold", "Noto Sans Telugu Bold",
+                "Noto Sans Thaana Bold",
+                "Noto Sans Thai UI Bold", "Noto Sans Thai Bold",
+                "Noto Sans Tibetan Bold", "Noto Sans Tifinagh Bold",
+                "Noto Sans Ugaritic Bold", "Noto Sans Vai Bold",
+                "Noto Sans Yi Bold",
+                "Noto Kufi Arabic Bold", "Noto Emoji Bold",
+                "Noto Sans UI Regular", "Noto Sans Regular",
```

Comment:
We duplicated the regular fonts list into the bold list because for the specific fonts we were using there were some that had regular glyphs but not bold. Is this also the case with Noto? If not, we can drop the duplicates.